module-apps/apps-common/locks/handlers/PhoneLockHandler.hpp

```diff
@@ -20,6 +20,13 @@ namespace locks
 
     class PhoneLockHandler
     {
+      public:
+        enum class ReasonForRequest
+        {
+            Default = 0, // For default behaviors
+            UnlockForMTP
```

Comment:
I'm wondering if the name is proper - AFAIU this case is when you connect the phone to the PC and MTP wants to unlock the phone - so maybe it should be named `MTPUnlock` or something?